### DISCUSSION TITLE
Morse Runner 1.80 - Field Day Contest prototype

### DISCUSSION THREAD

**Discussion by w7sst** [w7sst/MorseRunner#46] [Announcements]
## Summary
This is the first release of the Morse Runner Community Edition. This release contains the initial FD prototype. This version contains:
- A contest selection drop-down list allowing a specific contest to be selected.
- Supported contests include CWOPS CWT, CQ WPX, ARRL Field Day, NCL NAQP, and the original HST Contest.
- The contest exchange is specified in a new Exchange Field. The exchange setting for each contest is retained between contests.

## Join Our Community
You are invited to join our community effort. For more information, please visit the [Morse Runner Community Edition](https://github.com/w7sst/MorseRunner#readme) project.

Our goal is to create program you love to use, so please help by leaving any feedback you would like to share on our [discussions page](https://github.com/w7sst/MorseRunner/discussions/categories/morse-runner-1-80).

## Installation
Uncompress the zip file to any folder and run `MorseRunner.exe`.

## Release Details
This release contains cumulative changes from several authors.

### Version 1.80 (Oct 2022)
  - Added multi-contest support (Coded by W7SST)
  - Added ARRL Field Day contest (Coded by W7SST)
  - Added NCL NAQP contest (Coded by W7SST)

### 1.71a (CT7AUP) Nov 2021
  - CWOPS CWT Contest
  - CW RX Min Speed.
  - CW RX Max Speed.
  - NR number of digits.

### 1.70 (BG4FQD) Aug 25, 2016
  - Adjust UI, support windows scheme.
  - Support showing callsign infomation, You can modify it in the "ARRL.LIST"
  - Disable hot key: "\" to prevent pressing by carelessness.
  - Some other bugs fixing.

### 1.69 (BG4FQD) Jul 16, 2016
  - Add "Hi-Score web page" server in MorseRunner.ini.
  - Change default Font to Cleartype 'segoe ui', 'Consolar';
  - Change string to Unicode, Building with Delphi 2010 sp3.

### 1.68 (VE3NEA) 2016
  - The last official release from Alex, VE3NEA

## What's Changed
* Issue #18 - bug fix - CWOPTS.isnum() incorrectly considered an empty string as valid by @w7sst in https://github.com/w7sst/MorseRunner/pull/19
* Issue #16 - bug fix - last char of last field is dropped by TCWOPS.De… by @w7sst in https://github.com/w7sst/MorseRunner/pull/17
* Issue #14 - Add some comments to CWT code in SaveQso(). by @w7sst in https://github.com/w7sst/MorseRunner/pull/15
* Issue #21 - rework ct7aup's Rx CW Speed adjustment by @w7sst in https://github.com/w7sst/MorseRunner/pull/22
* Resolves #23 - rework - move digit and CW Rx settings I/O into Ini.pas; code simplification by @w7sst in https://github.com/w7sst/MorseRunner/pull/26
* Fixes #27 - restore whitespace differences by @w7sst in https://github.com/w7sst/MorseRunner/pull/28
* Integrate Ve3nea-normalized branch into our main branch by @w7sst in https://github.com/w7sst/MorseRunner/pull/30
* Merge fixes for Issue #31, #32, #33, #34 into main branch by @w7sst in https://github.com/w7sst/MorseRunner/pull/35
* Merge FD Prototype by @w7sst in https://github.com/w7sst/MorseRunner/pull/37
* initial changes for 1.80 release notes/Readme.txt file by @w7sst in https://github.com/w7sst/MorseRunner/pull/40

## New Contributors
* @w7sst made their first contribution in https://github.com/w7sst/MorseRunner/pull/5
* @scotthibbs made their first contribution in https://github.com/w7sst/MorseRunner/pull/8

**Full Changelog**: https://github.com/w7sst/MorseRunner/commits/v1.80

<hr /><em>This discussion was created from the release <a href='https://github.com/w7sst/MorseRunner/releases/tag/v1.80'>Morse Runner 1.80 - Field Day Contest prototype</a>.</em>

**Comment by scotthibbs** [comment]
My feedback so far: 

I submitted issues for the menu items that need reworked and for renaming the splash screen. 

I'm in love with this program! Great work!

**Reply by w7sst** [reply]
Thank you Scott. I appreciate your contributions and support as well.

And thank you for your introductory [Morse Runner Community Edition](https://www.youtube.com/watch?v=Y1kHqJRytMM) YouTube video. Great work!
